src/components/list/List.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Content from '../content';
+
+const List = ({ nodeType, content, entryMap, keyPrefix, level }) => {
+    const Tag = nodeType === 'ordered-list' ? 'ol' : 'ul';
+
+    return (
+        <Tag data-type={ nodeType }>
+            { content.map((item, index) => (
+                <li data-type="list-item" key={ `${keyPrefix}-item-${index}` }>
+                    <Content data={ { content: item.content } } entryMap={ entryMap } level={ level + 1 } />
```

Comment:
[This](https://github.com/moxystudio/react-contentful-rich-text/pull/1/files#r417415723) also applies here.